src/components/Events/EventDetail/EventDetail.tsx

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from '@tanstack/react-query'
+import fetchEvent from 'api/fetch-event'
+const EventDetail = ({ eventId }: IProps) => {
+  const eventDetail = useQuery(['todos'], () => fetchEvent({ id: eventId }))
+
+  return <p>{eventId}</p>
```

Comment:
If you want to easily check what is coming to the frontend you can nice print the object using JSON.stringify like

```
return <pre>{JSON.stringify(eventDetail, undefined, 2)}</pre>
```

Reply:
ohh cool! thanx